UPGRADING.rst

```diff
@@ -10,6 +10,22 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#4011 Integrate monitoring SNS topic with Slack
+===============================================
+
+Operator
+~~~~~~~~
+
+Set up AWS Chatbot Slack client as `described
```

Comment:
The timing could be specified more precisely. You should lead with when this should happen. Currently your saying: "Do B. Then do C just before A." That should be "Just before A, do B and C."